packages/itwinui-css/src/tree/tree.scss

```diff
@@ -34,7 +34,7 @@ $iui-expander-button-width: calc(var(--iui-component-height-small) + 2px);
   min-inline-size: max-content;
 
   &:focus-visible {
-    outline: none;
+    outline: none !important;
```

Comment:
ExpandableBlock and actionable Tag have a double outline that needs to be fixed.

<img width="173" height="116" alt="Screenshot (ExpandableBlock)" src="https://github.com/user-attachments/assets/eb250c2e-7454-49b3-928b-bd2b4c43d0b6" />

<img width="157" height="77" alt="Screenshot (Tag with close button)" src="https://github.com/user-attachments/assets/fea140cc-318a-4721-ba5e-f9424f252a93" />

We'll probably need to add `outline: none !important` to `iui-link-action`.